src/index.ts

```diff
@@ -28,8 +34,10 @@ export interface pdfFile {
 export async function* generatePDF(
   browser: Browser,
   files: pdfFile[],
-  options?: PDFOptions
-): AsyncIterableIterator<Buffer> {
+  options: PuppeteerPDFOptions
```

Comment:
Si possible et pas de tslint error
```suggestion
  options: PuppeteerPDFOptions = {}
```

Ce qui évite de mettre un objet vide à l'appel de la fonction dans tes tests

Reply:
Je suis parti sur le fait de repasser options en optional, ça fait plus sens dans la mesure où `page.pdf` va aussi prendre ces mêmes options en optional